web/src/main/webapp/WEB-INF/config-security/config-security.properties

```diff
@@ -19,14 +19,14 @@ ldap.base.dn.pattern=uid={0},${ldap.base.search.base}
 # Define if groups and profile information are imported from LDAP. If not, local database is used.
 # When a new user connect first, the default profile is assigned. A user administrator can update
 # privilege information.
-ldap.privilege.import=false
+ldap.privilege.import=true
 
 # Define if LDAP groups should be create in catalog
 # database if they do not exist.
 ldap.privilege.create.nonexisting.groups=true
 
 # Define if users should be saved in the LDAP
-ldap.privilege.create.nonexisting.users=false
+ldap.privilege.create.nonexisting.users=true
```

Comment:
Same here. When you have an organization with thousand of LDAP users and only 20 using the catalog, it may be better to set it to false by default ? as this options will transfert all LDAP users in the catalog if I understand it ?